receiver/wavefrontreceiver/receiver.go

```diff
@@ -55,7 +55,9 @@ func (r *metricsReceiver) Start(ctx context.Context, host component.Host) error
 		},
 	}
 
-	carbonReceiver, err := fact.CreateMetrics(ctx, r.set, carbonCfg, r.nextConsumer)
+	set := r.set
+	set.ID = component.NewIDWithName(fact.Type(), r.set.ID.String())
```

Comment:
So we are replacing `wavefront` with `carbon` here. I'm good with this. However, would it be better to wrap the `fact` and make `Type()` return `wavefront` instead?

Reply:
Not sure I understand, fyi we do exactly this for simpleprometheus, so if you have a better solution we should implement there as well.

Reply:
Ok, let's merge this one to unblock the tests consistently